src/containters/Home/components/DiscountsList/index.js

```diff
@@ -0,0 +1,159 @@
+import React from 'react';
+import './index.scss';
+import Slider from 'react-slick';
+import 'slick-carousel/slick/slick.css';
+import 'slick-carousel/slick/slick-theme.css';
+import { IoIosArrowForward, IoIosArrowBack, IoMdAperture } from 'react-icons/io';
+import { TiFlash } from 'react-icons/ti';
+import { IconContext } from 'react-icons';
+import { Link } from 'react-router-dom';
+import { Line } from 'rc-progress';
+import Rate from 'rc-rate';
+import 'rc-rate/assets/index.css';
+
+export const DiscountsList = (props) => {
+
+    function NextArrow(props) {
+        const { onClick } = props;
+        return (
+            <div className="next_arrow"
+                onClick={onClick}
+            >
+                <IoIosArrowForward />
+            </div>
+        );
+    }
+
+    function PrevArrow(props) {
```

Comment:
- Please extract this 2 components and the 2 other ones in category list to a reusable separate 2 component and reuse it in both places